Students-Toolkit.md

```diff
@@ -0,0 +1,67 @@
+  
+The programme should not be only on coding. Could be speakers or social media contributors in the same way like the CNCF Ambassador Program https://github.com/cncf/ambassadors.
+  
+## Advantages
+- Helps CNCF in attracting college students to contribute to their projects.
+- Helps CNCF in attracting college students to use cloud native technologies.
+- Helps CNCF projects to test new features to speed their release and testing times.
+- Involve Students in programs like Community Bridge or Google Summer of Code to contribute with CNCF projects.
+- Open a space for students to get jobs related to cloud native technologies.
+- Create a space for Universities contribution by doing research with their students using CNCF projects.
+- Create a space for innovation and new ideas where universities, students and CNCF can get involved.
+- Promote diversity and inclusion in the way that universities and students in other countries can contribute.
+  
+## To be done
+- List CNCF projects that could be interesting in this student kit.
+- Define the minimum skills to develop in potential students contributors
+- Define the steps to get involved students into participate into GSoC related to CNCF projects
+- List professor, universities that can be interested in this program.
+- Generate a vault of projects, features of issues that the students/students can be involved in.
+- Assigning the students to the respective mentors based on the projects.
+- Define deadlines for testing ideas, issues related to GSoC
+- Periodic evaluation of students on the progress of the projects by the mentors.
+- A Certificate and a small token of appreciation on successful completion of the project.
+- Ambassadors to publicise the mentors programme and help students with a smooth entry in the project.
+- If possible create a CNCF conference for students to promote students participation
+- Gather students opinions about this kind of opportunities to enrich this student kit
+- Work together with universities to update their syllabus in the way to include cloud native theory in their courses.
+- Provide courses or small guides to create a quickstart to contribute to CNCF projects.
+  
+## Conducting CNCF Webinars and conferences in Colleges
+As part of the mentor programme, conducting CNCF webinars and conferences in college helps CNCF in attracting more students and there-by increasing the user base and also creating an awareness of what CNCF projects are and its benefits.
+  
+## Advantages
+- Generate Ideas and Innovation related to cloud native technologies in specific CNCF projects.
+- Create a sustainable way to test, innovate and contribute together with the industry and the academy.
+- Increase in developer/contributor base for CNCF projects.
+- A ambassador program recognition for students for their efforts.
```

Comment:
Sorry, what do you mean here?

Reply:
I think that its the same that i was talking in the beginning of the document so i deleted that part.